cms/djangoapps/contentstore/tests/test_courseware_index.py

```diff
@@ -1167,95 +1167,91 @@ def _group_conf_url(self, cid=-1):
 
     def _html_group_result(self, html_unit, content_groups):
         """
-        Return call object with arguments and content group for html_unit.
+        Return object with arguments and content group for html_unit.
         """
-        return call(
-            'courseware_content',
-            {
-                'course_name': unicode(self.course.display_name),
-                'id': unicode(html_unit.location),
-                'content': {'html_content': '', 'display_name': unicode(html_unit.display_name)},
-                'course': unicode(self.course.id),
-                'location': [
-                    unicode(self.chapter.display_name),
-                    unicode(self.sequential.display_name),
-                    unicode(html_unit.parent.display_name)
-                ],
-                'content_type': 'Text',
-                'org': self.course.org,
-                'content_groups': content_groups,
-                'start_date': datetime(2015, 4, 1, 0, 0, tzinfo=tzutc())
-            }
-        )
+        return {
+            'course_name': unicode(self.course.display_name),
+            'id': unicode(html_unit.location),
+            'content': {'html_content': '', 'display_name': unicode(html_unit.display_name)},
+            'course': unicode(self.course.id),
+            'location': [
+                unicode(self.chapter.display_name),
+                unicode(self.sequential.display_name),
+                unicode(html_unit.parent.display_name)
```

Comment:
I realize you just moved this code over, but just wondering -- aside from `location`, aren't these XBlock fields already unicode? Because if they are, it seems redundant to cast them to unicode again. And if they're utf-8 bytestrings for some reason, then this will break while trying to decode because it'll assume ASCII by default.